services/company_graph_store.py

```
@@ -1154,19 +1176,43 @@ async def list_companies(
+    def _website_from_row(row) -> Website | None:
+        """Reconstruct a Website from a SQLite row, preferring the full JSON
+        blob (which preserves inferred_stack / detected_systems) and falling
+        back to the scalar columns for rows written before the blob existed."""
+        if not row:
+            return None
+        d = dict(row)
+        blob = d.get("data")
+        if blob:
+            try:
+                return Website.model_validate_json(blob)
+            except Exception as exc:  # corrupt/legacy blob — fall back to columns
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)